src/common/plugin.ts

```diff
@@ -55,35 +49,41 @@ export abstract class Plugin implements IPlugin {
 
     const response = this.container.channelService.hasPermission(channelName, this.permission);
     if (!response) {
-      const rooms = Object.values(Constants.Channels[this.permission]);
-      const totalChannels = rooms.length;
-      rooms.splice(3);
+      const channels = Object.values(Constants.Channels[this.permission]);
+      const totalChannels = channels.length;
+      channels.splice(3);
```

Comment:
what happens if length<3?

Reply:
it just returns what it can. Assuming you are talking about the splice